File: Backend/Guts.Web/ClientApp/app/models/exercisesummary.model.ts
export class ExerciseSummaryModel {
    public exerciseId: number;
    public number: number;
    public numberOfTests: number;
    public numberOfPassedTests: number;

    constructor() {
        this.exerciseId = 0;
        this.number = 0;
        this.numberOfTests = 0;
        this.numberOfPassedTests = 0;
    }
}

File: Backend/Guts.Api/Models/ExerciseSummaryModel.cs
namespace Guts.Api.Models
{
    public class ExerciseSummaryModel
    {      
        public int ExerciseId { get; set; }
        public string Code { get; set; }
        public int NumberOfTests { get; set; }
        public int NumberOfPassedTests { get; set; }
        public int NumberOfFailedTests { get; set; }
        public int NumberOfUsers { get; set; }
    }
}